src/proxy/inapp/inapp_full_proxy.go

```diff
@@ -0,0 +1,188 @@
+package app
```

Comment:
This file and the test file use different package names. 
Should probably be inapp. Doesnt compile.